db/migrate/20160412133816_create_categories.rb

```diff
@@ -0,0 +1,9 @@
+class CreateCategories < ActiveRecord::Migration
```

Comment:
This table is not necessary as the string could also have been stored in the categorizations table. But this is also fine and is probably more railsy.